container/container_windows.go

```diff
@@ -125,11 +126,8 @@ func (container *Container) TmpfsMounts() ([]Mount, error) {
 	return mounts, nil
 }
 
-// UpdateContainer updates configuration of a container
+// UpdateContainer updates configuration of a container. Callers must hold a Lock on the Container.
```

Comment:
The places where this is called are holding locks in broader contexts (for more operations), it made sense to let callers decide when to lock.